drivers/counter/Kconfig.mcp7940n

```diff
@@ -1,4 +1,4 @@
-# Copyright (c) 2021 Laird Connectivity
```

Comment:
Curious from a copyright perspective, do normally change a backward copyright on a name change instead of adding a line item for the new company name? Again, just a question for folks that are more knowledgeable about this.

Reply:
thanks, I am curious about this as well. I can drop all the copyright changes if needed.

Reply:
I would leave the existing copyrights notices and potentially add new ones for Ezurio starting from 2024 next to them.

Reply:
Changing existing to the current company name works for me and how I've always seen it done, doesn't really make sense to have a copyright to something that doesn't exist

Reply:
Thanks for the feedback. I dropped all modifications and added new where it made sense. This simplifies the changes and keeps a nice history.